### DISCUSSION TITLE
Help me understand how i can trust this version of GCC?

### DISCUSSION THREAD

**Discussion by parlortrickss** [Welding-Torch/installc#3] [Q&A]
I like what you've done here, it has made using C easy on my work Windows laptop. I am only a hobbyist but i do have concerns around trusting this custom build.

It won't stop me using it, but i am more interested in how i could verify that this setup is just a pre-compiled GCC you supplied. How do i know what changes you made to make this work if any?

**Comment by Welding-Torch** [comment]
I'm not entirely sure how to answer your question. Install C is fundamentally just an Inno Setup Script that uses WinLibs(Open Source), and MinGW-w64(Open Source). This project is also open source. If you absolutely need to make sure that the Install C installer is 1:1, then you can read [CONTRIBUTING.md #Code](https://github.com/Welding-Torch/installc/blob/main/CONTRIBUTING.md#code) and build the installer on your own system. 

If you can recommend any alternatives to improve transparency, I'm open to it. Right now, I'm considering setting up GitHub Actions to run the script and build Install C automatically whenever I push an update to this repo.
